apps/web/src/config/paymaster.ts

```diff
@@ -93,6 +96,17 @@ export const paymasterInfo: {
   },
 }
 
+/**
+ * Contracts that the paymaster is allowed to interact with
+ */
+export const PAYMASTER_CONTRACT_WHITELIST = [
+  getUniversalRouterAddress(ChainId.ZKSYNC), // Universal Router on zkSync
+  getAddressFromMap(addresses.zkSyncAirDrop, ChainId.ZKSYNC), // ZKSync AirDrop
+
+  // Token addresses for Approval
+  ...paymasterTokens.map((token) => token.wrapped.address),
```

Comment:
Is it gonna block the approval of tokens outside this list?

Reply:
resolving this